### ISSUE TITLE
Crash when using ${{ }} syntax with `|` in yaml

### ISSUE THREAD

**Issue by dapirian** [rhysd/actionlint#128]
workflow yaml file is:
```
name: Pull Request
on: pull_request

jobs:
  some_job:
    steps:
      - run: |
          echo ${{ yadayada }}
```

stderr is:

```
panic: runtime error: slice bounds out of range [220:14]

goroutine 1 [running]:
github.com/rhysd/actionlint.(*Error).getIndicator(0xc00017a040, {0xc0001599f2, 0x2573})
  /home/runner/work/actionlint/actionlint/error.go:133 +0x1ef
github.com/rhysd/actionlint.(*Error).GetTemplateFields(0xc00017a040, {0xc000116000, 0x126cf40, 0xc0000a5a40})
  /home/runner/work/actionlint/actionlint/error.go:66 +0x57
github.com/rhysd/actionlint.(*ErrorFormatter).PrintErrors(0xc0000d8180, {0x1265ea0, 0xc0000a0008}, {0xc00017a280, 0x7, 0x2574}, {0xc000116000, 0x2573, 0x2574})
  /home/runner/work/actionlint/actionlint/error.go:245 +0xd4
github.com/rhysd/actionlint.(*Linter).LintFile(0xc0000d8180, {0x205929e6a, 0x19}, 0x1219c1c)
  /home/runner/work/actionlint/actionlint/linter.go:391 +0x3f4
github.com/rhysd/actionlint.(*Linter).LintFiles(0x1265ea0, {0xc0000a4070, 0xc0000d8080, 0xc0000b2258}, 0x0)
  /home/runner/work/actionlint/actionlint/linter.go:265 +0x337
github.com/rhysd/actionlint.(*Command).runLinter(0xc0001092c0, {0xc0000a4070, 0xc0000a65f8, 0x1}, 0x7, 0x0)
  /home/runner/work/actionlint/actionlint/command.go:103 +0x150
github.com/rhysd/actionlint.(*Command).Main(0xc0001092c0, {0xc0000a4040, 0x4, 0x4})
  /home/runner/work/actionlint/actionlint/command.go:176 +0x67d
main.main()
  /home/runner/work/actionlint/actionlint/cmd/actionlint/main.go:15 +0xd9
```

All of the conditions for this crash are not clear to me. In one workflow I have `on: [pull_request, workflow_call, workflow_dispatch]` and if I simply remove the `workflow_call` from the list, actionlint doesn't crash. When I tried to turn that into a minimal reproducible workflow yaml, I found that using yaml `|` syntax combined with GH-action `${{ }}` syntax in some (common) situations also causes a crash, though I'm using that all over the place in other situations without crashing.
In any case, there are multiple crashes related to this.

**Comment by rhysd** [comment]
I tried the following steps but it didn't crash.

1. Copy your repro as `test.yaml`
2. Run `actionlint ./'test.yaml`

Is there any extra step?

And would you tell me the output from `actionlint -version`?

**Comment by dapirian** [comment]
Great, thanks!

This is on 1.6.10.

Oh sorry about that, I didn't think it mattered but it appears it's specific to passing `-format`, so this repros it:
```
actionlint -format '{{json .}}' .github/workflows/pr.yaml
```
Using the yaml above

**Comment by rhysd** [comment]
Thanks! I could reproduce this issue.

**Comment by rhysd** [comment]
I could add better fix at ed85253 and also added a regression test. This change will be release at v1.6.11.

**Comment by dapirian** [comment]
hey just wanted to confirm the crash is fixed IRL for me w/ 1.6.11 - thanks again! 👍
